lib/matplotlib/axes/_axes.py

```diff
@@ -5504,9 +5504,10 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
             which can be set by *filterrad*. Additionally, the antigrain image
             resize filter is controlled by the parameter *filternorm*.
 
-        alpha : scalar, optional
+        alpha : scalar or array_like, optional, default: None
             The alpha blending value, between 0 (transparent) and 1 (opaque).
-            This parameter is ignored for RGBA input data.
```

Comment:
I have dropped `This parameter is ignored for RGBA input data.` because it appears to be respected by the released `matplotlib==3.1.1`.